app/controllers/api/v1/tasks_controller.rb

```diff
@@ -59,7 +60,7 @@ def activities
       def task_params
         raise TypeError unless params.require(:data).require(:type) == 'tasks'
 
-        params.require(:data).require(:attributes).permit(%i(name x y description state))
```

Comment:
Can you set not first my_module_status on the `create` method also?  Or will fail with validation for check_status?